Modules/_asynciomodule.c

```diff
@@ -2296,14 +2297,8 @@ task_step(TaskObj *task, PyObject *exc)
         PyObject *et, *ev, *tb;
         PyErr_Fetch(&et, &ev, &tb);
         ot = _PyDict_Pop(current_tasks, task->task_loop, NULL);
-        if (ot == NULL) {
-            Py_XDECREF(et);
-            Py_XDECREF(tb);
-            Py_XDECREF(ev);
-            return NULL;
-        }
-        Py_DECREF(ot);
-        PyErr_Restore(et, ev, tb);
+        Py_XDECREF(ot);
+        _PyErr_ChainExceptions(et, ev, tb);
```

Comment:
I'm surprised this API is usable from extension module while `python3.def` doesn't include this.
Maybe, I don't understand what APIs can be used from extension on Windows...